playground/fibonacci_using_for.js

```diff
@@ -0,0 +1,26 @@
+function fibo (idx) {
+    let sumNum = 0;
+    
+        if (idx == 1 || idx == 2) {
+            return 1
+        } else if( idx >= 3) {
+            let curNum=1 ;
+            let bfNum=1 ;
+            for ( let i=1; i<=idx-2; i++){
+    
+                sumNum = bfNum + curNum;
+                bfNum = curNum;
+                curNum = sumNum;
+    
+            }
+            return sumNum;
```

Comment:
```suggestion
            for ( let i=1; i<=idx-2; i++){
                sumNum = bfNum + curNum;
                bfNum = curNum;
                curNum = sumNum;
            }
 
            return sumNum;
```

이렇게 하는 편이 가독성이 좋아보입니다.

Reply:
네 수정했습니다!